### PR TITLE
chore: clean up generated config and config types

### PR DESCRIPTION
**Pull request by VIKTORVAV99**
## Issue

We had some Any types, included data we didn't use, and had sub-optimal config generation.

## Description

This PR does 3 small things:

- It cleans up 2 Any types
- It removes the contributor array from a zone config if it don't have any contributors.
- It removes the capacity data from exchanges as that is unused as we use the data sent through the backend.

### Double check

- [x] I have run `pnpx prettier@2 --write .` and `poetry run format` in the top level directory to format my changes.

### CODE REVIEW

File: web/geo/types.ts [hunk 1]
```diff
@@ -21,13 +21,23 @@ export interface ZoneConfig {
...
     productionPerUnit?: string;
   };
-  [key: string]: any;
+  [key: string]:
+    | undefined
+    | string[]
+    | number[][]
+    | string
+    | boolean
+    | { [key: string]: string };
```
Comment (tonypls) [inline web/geo/types.ts:30]: Haha, at this point I'm not sure it's better than any 🤣

Comment (VIKTORVAV99) [reply web/geo/types.ts:30]: Agreed but at least ESLint don't complain 😅

**Review by tonypls — APPROVED** [review]
LGTM

File: web/geo/types.ts (hunk 1 above)
Comment (tonypls) [reply web/geo/types.ts:30]: We could add this somewhere 😆 
```
export interface Any {
  // Primitive types
  string: string;
  number: number;
  boolean: boolean;
  null: null;
  undefined: undefined;
  symbol: symbol;
  bigint: bigint;

  // Object types
  object: object;
  array: any[];
  tuple: [any, any];
  function: (...args: any[]) => any;

  // Union type
  union: string | number | boolean;

  // Intersection type
  intersection: { a: string } & { b: number };

  // Literal types
  stringLiteral: "literal";
  numberLiteral: 42;
  booleanLiteral: true;

  // Enum
  enum: "A" | "B" | "C";

  // Generic types
  generic: Array<any>;
  promise: Promise<any>;
  map: Map<any, any>;
  set: Set<any>;

  // Class
  class: { new (...args: any[]): any };

  // Recursive type
  recursive: Any;

  // Optional property
  optional?: any;

  // Index signature for additional properties
  [key: string]: unknown;
}
```

**Pull request merged**